IngressMaxFields.user.js

```diff
@@ -13,6 +13,9 @@
 // @match http://www.ingress.com/intel*
 // @grant none
 // ==/UserScript==
+/*global $:false */
+/*global map:false */
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: ["jQuery" is defined but never used](https://www.codacy.com/app/stefan-nygren/IITC-Ingress-Maxfields-Exporter/file/1750281813/issues/source?bid=3039347&fileBranchId=3045939#l18)